test-infra/prowjobs/cleanerupper/Dockerfile

```diff
@@ -11,13 +11,10 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-FROM golang:alpine
+FROM golang
 
-RUN apk add --no-cache git
-
-WORKDIR /tmp
-COPY main.go main.go
-RUN go get -d ./...
+WORKDIR /build
+COPY . .
```

Comment:
What does this do?

Reply:
Copies everything in the current directory to the working directory, its a bit odd here because of how the WORKDIR works but the build starts in a folder (in this case it has a checkout of the git repo in it). I then set the WORKDIR to /build, so all build actions take place in there. I then copy the starting directory to that working directory